src/components/VideoAsset.vue

```diff
@@ -15,28 +15,38 @@
     :should-wrap="!!deviceFrame"
     :device="deviceFrame"
   >
-    <video
-      ref="video"
-      :controls="showsControls"
-      :data-orientation="orientation"
-      :autoplay="autoplays"
-      :poster="normalisedPosterPath"
-      :muted="muted"
-      :width="optimalWidth"
-      playsinline
-      @loadedmetadata="setOrientation"
-      @playing="$emit('playing')"
-      @pause="$emit('pause')"
-      @ended="$emit('ended')"
-    >
-      <!--
-        Many browsers do not support the `media` attribute for `<source>` tags
-        within a video specifically, so this implementation for dark theme assets
-        is handled with JavaScript media query listeners unlike the `<source>`
-        based implementation being used for image assets.
-      -->
-      <source :src="normalizePath(videoAttributes.url)">
-    </video>
+    <div>
```

Comment:
Is the extra div needed because of the way that `ConditionalWrapper` works? Just curious. It would be nice to not add it, but I'm guessing that you already tried and found that it's necessary.

Reply:
Yes, I tried to do:
```html
<ConditionalWrapper
    ref="wrapper"
    :tag="!!deviceFrame ? DeviceFrameComponent : 'div'"
    :should-wrap="!!deviceFrame || !!alt"
    :device="deviceFrame"
  >
```
which in theory it would allow me to render a 'div' in case that we pass an `alt` text: the `tag` prop is only used when `shouldWrap` prop is true.

But testing without an `alt` text it returns the Vue error:
`Multiple root nodes returned from render function. Render function should return a single root node.`

Somehow it counts the span text as multiple root node, even when it doesn't render. 😵‍💫